src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: flex;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  .stars--4 &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
The selector '.stars--4 &__star:nth-child(-n + 4)' is incorrect. The '&' should be used to refer to the parent selector, but here it is used incorrectly. It should be '.stars--4 .stars__star:nth-child(-n + 4)' to correctly target the first four stars within an element with the 'stars--4' class.